main.tf

```diff
@@ -811,9 +811,9 @@ resource "aws_security_group" "this" {
   vpc_id      = var.vpc_id
 
   tags = merge(
+    { "Name" = local.security_group_name },
     var.tags,
     var.security_group_tags,
-    { "Name" = local.security_group_name },
```

Comment:
tags are merged from general to more specific, since the latter will overwrite the former if the same key exists. This is correct as its written today - if you wish to change the name, you can do so via `var.security_group_name`

Reply:
@bryantbiggs, yes I can change `var.security_group_name`, but that is a destructive action and I will need to recreate all of my security groups (I have imported them after this module added security group support).

I'd also be happy to leave `name` and `name_prefix` blank so that it is automatically generated by terraform, as I have done in the past. Do you have a suggestion for how to do this, it seems to me that `name` or `name_prefix` is required and I'd prefer not to set either.

Reply:
This module requires a name for the security group. Instead of importing existing security groups that conflict with the module implementation, you can let the module create net new security groups with the desired ingress/egress rules, and then remove your previous security groups created externally. This would not be disruptive since the two security groups would permit the same level of access